src/main/java/axoloti/utils/Preferences.java

```diff
@@ -200,6 +208,37 @@ public String getCurrentFileDirectory() {
         return CurrentFileDirectory;
     }
 
+    public IPlatform getPlatform(Patch p) {
+        IPlatform target = null;
+        if (Platform != null && !Platform.isEmpty()) {
+            try {
+                Logger.getLogger(Preferences.class
+                        .getName()).log(Level.INFO, "Attempting to load target {0}", Platform);
+                ClassLoader cl = Axoloti.class.getClassLoader();
+                Class c = cl.loadClass(Platform);
+                Object o = c.newInstance();
```

Comment:
I don't think the full generality of this String->Class loading mechanism is really necessary here. Especially because there isn't actually an alternative platform yet that would use this code. The valid platforms should be enumerated somewhere and mapped directly to the appropriate class instantiations. There shouldn't be the potential to instantiate some arbitrary class here that we don't know about. Like possibly in the future the Platform string would be something selected in preferences, etc.

Something as simple as:
```java
if(platform == "1337Platform") {
    return 1337Platform();
}
```
seems fine to me because the number of platforms is going to be small.